vars/supportedUpgradeFromVersions.groovy

```diff
@@ -1,8 +1,7 @@
 #!groovy
 
-def call(String git_branch, List base_versions_list) {
-    def clean_branch_name = git_branch.tokenize('-')[-1]
-    if (is_enterprise_version(clean_branch_name)) {
+def call(List base_versions_list, String new_scylla_repo) {
+    if (new_scylla_repo.contains('enterprise')) {
```

Comment:
why would we bother to still check this? we could simple know that for each test, we have specific `base_versions` we need to check for each SCT branch, no?

Reply:
we need to know how to filter the enterprise release once we run open source version, cause upgrade from enterpriase to OSS isn't supported